app/views/teachers/edit.html.erb

```diff
@@ -1,5 +1,15 @@
-<%= provide(:h1, "#{@status} Edit #{@teacher.full_name}") %>
-
+<div class='container'>
+  <div class='row'>
+    <div class='col'>
+      <h1> <%= "#{@status} Edit #{@teacher.full_name}" %> </h1>
+    </div>
+    <% if @is_admin or @teacher.validated? %>
+      <div class='col' style="position:relative;">
+        <%= button_to "Resend Welcome Email", { action: "resend_welcome_email", id: @teacher.id }, {:class => "btn btn-primary", :style => "width:auto;position:absolute;right:0;margin:auto;bottom:15%" } %>
```

Comment:
Would it be possible to create the style in a css file instead of doing inline?

Reply:
Just added a separate CSS file. Also: I should've noticed this earlier but I updated the teachers#show page with a few functions that were already being used in _teacher.erb. Should make things a little cleaner (displays 'CSP Teacher' instead of 'Csp_Teacher' for example)